### ISSUE TITLE
Performance of cheetah-run task

### ISSUE THREAD

**Issue by jiaweihhuang** [matsuolab/BREMEN#6]
Hi,

I tried to run deployment-efficient experiments to reproduce the results reported in the paper with the following command:
```
python recursive.py --env cheetah_run --exp_name recursive_example --sub_exp_name BREMEN_demo --param_path configs/params_cheetah_run_offline.json --bc_init --random_seeds 0 --target_kl 0.1 --max_path_length 250 --gaussian 0.1 --const_sampling
```
After the training is finished, I observed the following evaluation results:
```
---------------------------
| Iteration    | 399      |
| TotalSamples | 850000   |
| episode_max  | 745      |
| episode_mean | 741      |
| episode_min  | 735      |
---------------------------
```
However, according to Fig 2 in the original paper (https://arxiv.org/pdf/2006.03647.pdf), the performance of HalfCheetah should be around 6000, which is quite different from the evaluation results.

I wonder the parameter setting specified in the command above is the same as the setting of experiments in this paper? If not, could you let me know which hyper-parameter should be modified in order to reproduce the results reported in the paper? Or maybe there are some other reasons for this performance gap?

Thanks a lot!

**Comment by frt03** [comment]
@Leonardo-H 

`cheetah_run` uses a different reward function (based on DM Control) from `HalfCheetah` (based on Open AI Gym MuJoCo). If you reproduce the main results in the paper, you should use `HalfCheetah` environments (also `cheetah_run` result is described in Appendix).

**Comment by jiaweihhuang** [comment]
Got it. Thanks for your reply!
